staging/src/k8s.io/apiserver/pkg/server/filters/priority-and-fairness.go

```diff
@@ -97,6 +102,8 @@ func WithPriorityAndFairness(
 			served = true
 			innerCtx := context.WithValue(ctx, priorityAndFairnessKey, classification)
 			innerReq := r.Clone(innerCtx)
+			w.Header().Set(responseHeaderMatchedPriorityLevelConfigurationUID, string(classification.PriorityLevelUID))
```

Comment:
Actually, I think it's probably better to use the names rather than the UIDs here?

Reply:
We discussed this in a previous meeting.  The concern is that this leaks information that might be sensitive.  It is somewhat like allowing listing of these objects without permission from RBAC.

Reply:
Oh, right, I remember even pointing that out myself. Thanks :)

Reply:
marking this resolved as it's addressed